apis/src/functions/pwa.rs

```diff
@@ -0,0 +1,26 @@
+#[cfg(feature = "ssr")]
+use actix_web::{get, HttpResponse, Responder, web};
+use walkdir::WalkDir;
+
+#[cfg(feature = "ssr")]
+#[get("/pwa-cache")]
+pub async fn cache(site_root: web::Data<String>) -> impl Responder {
```

Comment:
@frisoft You could use https://docs.rs/lazy_static/latest/lazy_static/ so that we only need to compute the cache once. But I am happy to roll it out as is and then when it's getting too slow we can intervene 👍

Reply:
I'd merge this one and create another PR to introduce the lazy_static.OK?